packages/components/dropdown/src/dropdown-item-impl.vue

```diff
@@ -1,10 +1,10 @@
 <template>
-  <div
+  <li
     v-if="divided"
     :class="ns.bem('menu', 'item', 'divided')"
     v-bind="$attrs"
-  ></div>
-  <div
+  ></li>
```

Comment:
Maybe we should keep the original `div` but add `role="separator"` to it.

Reply:
> Maybe we should keep the original `div` but add `role="separator"` to it.

But div should not be child element of ul, right?

Reply:
Yes, `div` can't be a child of `ul`.  
See: [\<ul\>: The Unordered List element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul)

> Permitted content [of ul] | Zero or more \<li\>, \<script\> and \<template\> elements.

Reply:
> Yes, div can't be a child of ul.
See: [<ul>: The Unordered List element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul)

I see, should we add `role="separator"` for the `li` element?

Reply:
I don't think that it's needed as a `li` is already a semantic element and will be declared as an item.

In the `<el-dropdown-item>`, there is an *attribute* `divider`, but I'm not sure to understand why it would be needed as it seems to do nothing when I test it in the fiddle.

I'm pretty new to *element+* so it might be interesting to wait for a more experienced opinion.